saleor/plugins/avatax/plugin.py

```diff
@@ -373,29 +372,64 @@ def calculate_checkout_line_total(
 
         return base_total
 
-    def _calculate_order_line_unit(self, order_line):
+    def calculate_checkout_line_unit_price(
+        self,
+        checkout: "Checkout",
+        checkout_line: "CheckoutLine",
+        address: Optional["Address"],
+        discounts: Iterable["DiscountInfo"],
+        variant: "ProductVariant",
+        collections: List["Collection"],
+        channel: "Channel",
+        channel_listing: "ProductVariantChannelListing",
+        previous_value: TaxedMoney,
+    ):
+        if not checkout_line.variant.product.charge_taxes:
```

Comment:
```suggestion
        if not variant.product.charge_taxes:
```
or better solution to pass the product to this function.

Reply:
We can pass `CheckoutLineInfo` too.